flowkit_jwt_generator/flowkit_jwt_generator/jwt.py

```diff
@@ -0,0 +1,213 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import datetime
+import os
+import uuid
+from json import JSONEncoder
+from typing import Dict, List, Union, Callable
+from datetime import timedelta
+import click
+import jwt
+import pytest
+import requests
+
+
+permissions_types = {"run": True, "poll": True, "get_result": True}
+
+
+def generate_token(
+    username: str,
+    secret: str,
+    lifetime: datetime.timedelta,
+    claims: Dict[str, Union[Dict[str, bool], List[str]]],
+) -> str:
+    """
+
+    Parameters
+    ----------
+    username : str
+        Username for the token
+    secret : str
+        Shared secret to sign the token with
+    lifetime : datetime.timedelta
+        Lifetime from now of the token
+    claims : dict
+        Dictionary of claims the token will grant
```

Comment:
Can we add a simple example to the docstring of what this dictionary looks like?